cscs-checks/microbenchmarks/osu/osu_tests.py

```diff
@@ -1,17 +1,16 @@
-import os
-import itertools
+import reframe as rfm
 import reframe.utility.sanity as sn
 
-from reframe.core.pipeline import RegressionTest
 
-
-class AlltoallBaseTest(RegressionTest):
-    def __init__(self, name, **kwargs):
-        super().__init__(name,
-                         os.path.dirname(__file__), **kwargs)
+@rfm.parameterized_test([['production']])
+class AlltoallTest(rfm.RegressionTest):
+    def __init__(self, tags):
```

Comment:
I didn't mean it like that. The tests should take a single *string* argument of whether it is production or not. Then you set the tags as usual:

```python
@rfm.parameterized_test(['production'])
class AlltoallTest(rfm.RegressionTest):
    def __init__(self, variant):
        self.tags = {variant}
```